.github/workflows/ci.yml

```diff
@@ -8,6 +8,21 @@ on:
     branches: ["master", "main"]
 
 jobs:
+  linter:
+    runs-on: ubuntu-latest
+    name: linter/ruby
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
```

Comment:
usually in gems you test against multiple ruby versions, you can check it out in https://github.com/rootstrap/yaaf/blob/master/.github/workflows/ci.yml#L12